test/TestProjects/MgmtSingleton/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -5,10 +5,20 @@
 
 #nullable disable
 
+using Azure.ResourceManager.Core;
+
 namespace MgmtSingleton
 {
     /// <summary> Extension methods for convenient access on SubscriptionOperations in a client. </summary>
     public static partial class SubscriptionExtensions
     {
+        #region Get SingletonResource3Operationes operation
+        /// <summary> Gets an object representing a SingletonResource3Operations along with the instance operations that can be performed on it. </summary>
+        /// <returns> Returns a <see cref="SingletonResource3Operations" /> object. </returns>
+        public static SingletonResource3Operations GetSingletonResource3Operationes(this SubscriptionOperations subscriptionOperations)
```

Comment:
There are a few singleton resources all named the same and only 1 shows up here.  Can we name them with more descriptive names?  Such as is the reason only this one is here is because its the only one with a subscription parent?  Maybe name it SubscriptionParentSingleton?  Similarly maybe we can name the others so its clear what is expected and why its expected?